components/hacbs/jvm-build-service/base/kustomization.yaml

```diff
@@ -4,6 +4,26 @@ resources:
 patchesStrategicMerge:
 - self-binding.yaml
 
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: IMAGE_TAG
+          value: b911059a17e1efdbc2ddb496604e673d0c491c43
```

Comment:
If I ever caught how the appstudio staging bot knows what fields in this file it needs to update when jvm build service repo PRs merge, I've lost it.

wherever that intelligence is, does it need additional modification to update this field as well?

thanks

Reply:
https://github.com/redhat-appstudio/jvm-build-service/blob/main/.tekton/controller-push.yaml#L23-L26 , unfortunately it can use only basic linux commands, so sed is useful but it's not using context -> hard to apply when you have multiple lines.

Reply:
gotcha / bummer

fingers crossed we can munge some bash that works

thanks for the pointer

Reply:
it is available